client/me/purchases/titles.js

```diff
@@ -56,7 +56,7 @@ Object.defineProperties( titles, {
 		get: () => i18n.translate( 'Payment Methods' ),
 	},
 	vatDetails: {
-		get: () => i18n.translate( 'VAT Details' ),
+		get: () => i18n.translate( 'Business Purchase ID Details' ),
```

Comment:
Should this be `Business Tax ID details` (or `Business Tax ID Details` maybe) to match the wording in the other text?

Reply:
I think lowercase makes more sense, since 'details' isn't really part of a title.